<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calkuladora</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Diseño/logo2.png</normaloff>:/Diseño/logo2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QPushButton" name="salirPB">
      <property name="text">
       <string>SALIR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="logo1LB">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Recursos.qrc">:/Diseño/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>471</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="1">
          <widget class="QPushButton" name="raizLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>RAÍZ</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="logaritmicasLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>LOGARITMICAS</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="coshiperbolicoLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>COS. HIPER.</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="tanLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>TAN</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="moduloLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>MÓDULO </string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="cosLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>COS</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QPushButton" name="senohiperbolicoLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>SEN. HIPER.</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="tanhiperbolicoLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>TAN. HIPER.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="senLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>SEN</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="factorialLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>FACTORIAL</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="sumaLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>SUMA</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="divisionLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>DIVISIÓN</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="potenciaLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>POTENCIA</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="pushButton_16">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>V. ABSOLUTO</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="restaLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>RESTA</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="multiplicacionLB">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>MULTIPLICACIÓN</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>301</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Stencil&quot;;</string>
        </property>
        <property name="text">
         <string>CALCULADORA EN ENSABLADOR</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="operacion1LB">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>86</width>
          <height>19</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 12pt &quot;Stencil&quot;;</string>
        </property>
        <property name="text">
         <string>OPERACIÓN</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>261</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QLineEdit" name="cantidad2LE"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="cantidad1LE"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>CANTIDAD 1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>CANTIDAD 2</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>140</y>
          <width>111</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="calcular1PB">
           <property name="text">
            <string>CALCULAR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="resultado1LB">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>RESULTADO</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="igualLB">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="regresar1PB">
           <property name="text">
            <string>REGRESAR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>131</width>
          <height>251</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="operacion2LB">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>OPERACION</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="calcular2PB">
           <property name="text">
            <string>CALCULAR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="resultado2LB">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>RESULTADO</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="igual2LB">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Stencil&quot;;</string>
           </property>
           <property name="text">
            <string>=</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="regresar2PB">
           <property name="text">
            <string>REGRESAR</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
